api/extensions/storage/oci_storage.py

```diff
@@ -0,0 +1,64 @@
+from collections.abc import Generator
+from contextlib import closing
+from flask import Flask
+from botocore.client import Config
+from botocore.exceptions import ClientError
+from extensions.storage.base_storage import BaseStorage
+import boto3
+import os
+
+
+class OCIStorage(BaseStorage):
+    def __init__(self, app: Flask):
+        super().__init__(app)
+        app_config = self.app.config
+        self.bucket_name = os.getenv('OCI_BUCKET_NAME')
```

Comment:
Please use `app.config` to fetch configurations, and declare configuration parameters in the `configs`. You can refer to other storage implementations for guidance. Thank you!

Reply:
hi @takatost ,i had modify it,please check it ,Thanks!